src/Entities/IssueDetails.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace ITBugTracking\Entities;
+
+use ITBugTracking\Entities\Issue;
+
+class IssueDetails extends Issue
+{
+    public array|null $comments;
+    public string $reporter;
+    public int $department;
+    public string $description;
+
+    public function jsonSerialize(): mixed
+    {
+        return [
+            'id' => $this->id,
+            'title' => $this->title,
+            'severity' => $this->severity,
+            'date_created' => $this->date_created,
+            'comment_count' => $this->comment_count,
+            'reporter' => $this->reporter,
+            'department' => $this->department,
+            'description' => $this->summary,
```

Comment:
Surely this is the `description`.

'Summary' on the `Issue` should *always* be the first 100 characters of the description. The `description` on the `IssueDetails` should *always* be all of the description. You should treat these things consistently.

Reply:
Matt's done this :)